src/client/game.cpp

```diff
@@ -3802,7 +3804,10 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats, f32 dtime,
 			(camera->getCameraMode() != CAMERA_MODE_THIRD_FRONT));
 #ifdef HAVE_TOUCHSCREENGUI
 	try {
-		draw_crosshair = !g_settings->getBool("touchtarget");
+		draw_crosshair = !g_settings->getBool("touchtarget") ||
+				(g_settings->getBool("use_crosshair") &&
```

Comment:
Isn't it better to locally cache these settings, instead of retrieving them every frame? I assume `Game::updateFrame` is run once every frame.

Reply:
I agree.